doc/source/notebooks/manual/advanced/ordinal_regression.ipynb

```diff
@@ -7,7 +7,212 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#make a one dimensional ordinal regression problem\n",
```

Comment:
Can you collect these steps into a function `generate_data(num_data)` so that a user knows he/she can skip it if not interested